src/classes/Integration/GravityForms.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/**
+ * Gravity Forms integration class
+ *
+ * @package notification
+ */
+
+namespace BracketSpace\Notification\Integration;
+
+/**
+ * GravityForm integration class
+ */
+class GravityForms {
+
+	/**
+	 * Replaces the default hooks for the new user notification
+	 *
+	 * @action notification/init
+	 *
+	 * @since  [ Next ]
+	 * @return void
+	 */
+	public function init() {
+		add_filter( 'gform_pre_send_email', [ $this, 'disable_admin_email' ], 10, 4 );
+		add_action( 'phpmailer_init', [ $this, 'stop_email' ] );
+	}
+
+	/**
+	 * Disable admin email
+	 *
+	 * @since [ Next ]
+	 *
+	 * @param array  $email          An array containing the email to address, subject, message, headers, attachments and abort email flag.
+	 * @param string $message_format The message format: html or text.
+	 * @param array  $notification   The current Notification object.
+	 * @param array  $entry          The current Entry object.
+	 *
+	 * @return array
+	 */
+	public function disable_admin_email( $email, $message_format, $notification, $entry ) {
+		if ( ( 'Admin Notification' === $notification['name'] && 'true' === notification_get_setting( 'integration/emails/new_user_to_admin' ) ) ) {
```

Comment:
This `'Admin Notification'` seems to be super shady. Are you sure this name is always going to be this string? I think we should find a better way to identify this email.

Plus the `$notification   The current Notification object.` is misleading. Is this our notification object? If it's an object then why you accept an array?